Makefile

```diff
@@ -14,17 +14,12 @@ s:
 
 # Start the server using forever
 sf:
-	$(BIN)/forever node ./index.js --max-old-space-size=512
+	$(BIN)/forever ./index.js --max-old-space-size=512
 
 # Run all of the project-level tests, followed by app-level tests
 test: assets
-	$(BIN)/mocha $(shell find api/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find api/apps/*/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find client/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find client/apps/*/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find client/apps/*/**/*/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find client/apps/*/**/*/**/test -name '*.coffee' -not -path 'test/helpers/*')
-	$(BIN)/mocha $(shell find client/apps/*/**/*/**/*/test -name '*.coffee' -not -path 'test/helpers/*')
+	$(BIN)/mocha $(shell find api -name '*.test.coffee')
+	$(BIN)/mocha $(shell find client -name '*.test.coffee')
```

Comment:
This change also clears the way to [co-locate your tests with your sources](https://github.com/artsy/emission/tree/master/src/lib/Components/Artist/__tests__) should you ever want to go that route.